projects/igniteui-angular/src/lib/expansion-panel/expansion-panel-header.component.ts

```diff
@@ -131,7 +131,7 @@ export class IgxExpansionPanelHeaderComponent {
      * ```
      */
     @Output()
-    public onInteraction = new EventEmitter<IExpansionPanelEventArgs>();
+    public onInteraction = new EventEmitter<IExpansionPanelInteractionEventArgs>();
```

Comment:
you should update the comment too, as it is pointing to the old event arguments type